hack/ssh.sh

```diff
@@ -14,11 +14,11 @@ ID="$2"
 shift 2
 
 IP=$(az vmss list-instance-public-ips -g $RESOURCEGROUP -n ss-master --query "[$ID].ipAddress" | tr -d '"')
-
+echo $IP
```

Comment:
why does this file change, out of interest?

Reply:
Because silent ssh bugs me so much as when it breaks and I don't know why. I have to go in, modify the file, add some debug messages, run again to check why it fails.  And later commit it by mistake with all other changes... :) Simple :D

Reply:
it only breaks for you ;-)